blacklisted-modules.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can we also use this to blacklist the old jurors registry?

Reply:
yup :)